.uncrustify.cfg

```diff
@@ -0,0 +1,240 @@
+# Configuration options for Uncrustify specifying the Mbed TLS code style.
```

Comment:
I'd prefer to copy that file from Mbed TLS instead of committing it into the development branch.

Reply:
Ok, can change that.

Reply:
I will need to add a new function to the `psa_crypto.py` script to allow for copying files from the root directory in order to achieve this since currently only things from sub-directories are copied.

Reply:
yes sure you can add one